src/einsteinpy/tests/test_symbolic/test_christoffel.py

```diff
@@ -98,3 +98,32 @@ def test_properties():
         assert False
     except Exception:
         return True
+
+
+def test_lorentz_transform():
+    # currently testing correct instance, proper theoretical tests needed
+    def get_lorentz_matrix():
+        list2d = [[0 for t1 in range(4)] for t2 in range(4)]
+        phi = symbols("phi")
+        list2d[0][0], list2d[0][1], list2d[1][0], list2d[1][1] = (
+            cosh(phi),
+            -sinh(phi),
+            -sinh(phi),
+            cosh(phi),
+        )
+        list2d[2][2], list2d[3][3] = 1, 1
+        return list2d
+
+    def get_tensor():
+        x, y, z, w = symbols("x y z w")
+        list2d = [[0 for t1 in range(4)] for t2 in range(4)]
+        list2d[0][0], list2d[0][1], list2d[1][0], list2d[1][1] = x, z, z, x
+        list2d[2][2], list2d[3][3] = y, y
+        return ChristoffelSymbols(
+            list2d, syms=(x, y, z, w), config="lu", parent_metric=None
+        )
```

Comment:
`ChristoffelSymbols` is a third order tensor while you declared a second order tensor. Anyway, I have a better solution to get an instance of this class. There are many predefined metrics and we can use one of them to get an instance of `ChristoffelSymbols`. In the `get_tensor()` function, do this.

```
metric = AntiDeSitter()
return ChristoffelSymbols.from_metric(metric)
```

To get `AntiDeSitter` kindly do `from einsteinpy.symbolic.predefined import AntiDeSitter` .
Let's see what happens